commands/tags.go

```diff
@@ -31,33 +31,36 @@ func Tags() *Command {
 			Short: "Display commands to manage tags",
 			Long: `The sub-commands of ` + "`" + `doctl compute tag` + "`" + ` manage the tags on your account.
 
-A tag is a label that can be applied to a resource (currently Droplets, images,
-volumes, volume snapshots, and database clusters) in order to better organize or
```

Comment:
The fact that only Droplets, images, volumes, volume snapshots, and database clusters are taggable resources seems like useful info that we should retain somewhere. Maybe in the `apply` command's help?

Reply:
You're absolutely right. I'll work that back in there.